### DISCUSSION TITLE
Newest Version installed but not in plug in.

### DISCUSSION THREAD

**Discussion by MavicaCameras** [WarmUpTill/SceneSwitcher#1112] [Q&A] [answered]
There was no errors and installed.
However do not see oy located under OBS.

**Comment by WarmUpTill** [comment]
Thanks for reporting the problem!

I am currently in the process of trying to identify which change caused to happen.
To work around this problem you can try installing the latest Microsoft Visual C++ Redistributable.

I will release a new version once I have a fix, which should be soon hopefully.

**Reply by WarmUpTill** [reply, marked as answer]
Can you please give the release 1.26.2 a try.
I _think_ this should solve the issue.

**Comment by MavicaCameras** [comment]
Thank You :) Yes it is on the Tool menu.
I have not tested it yet, But it is there.
Thank You very much for your help.
Have a Great Week.
